src/assets/styles/base/_base.scss

```diff
@@ -77,6 +77,17 @@ a[rel='external'] {
   }
 }
 
+a[rel='home'] {
+  border: 2px solid;
+  border-radius: 5px;
+  outline: none;
```

Comment:
Rather than setting `outline` to `none` I think we should set it to `transparent`. Here is a good explanation: https://benmyers.dev/blog/whcm-outlines/